Assets/Editor/Elements/ITrack.cs

```diff
@@ -0,0 +1,7 @@
+using UnityEngine.UIElements;
+
+public interface ITrack
```

Comment:
Thinking about names. I believe we have ITrack for the Runtime curves, gradients and boolean stuff we made. Maybe ITrackElement for the UI?